db/migrate/20220921095507_add_coupon_type_and_percentage_rate_to_coupons.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class AddCouponTypeAndPercentageRateToCoupons < ActiveRecord::Migration[7.0]
+  def change
+    change_table :coupons, bulk: true do |t|
+      t.integer :coupon_type, null: false, default: 0
+      t.decimal :percentage_rate, precision: 10, scale: 5
+      t.change :amount_cents, :bigint, null: true
+      t.change :amount_currency, :string, null: true
+    end
+
+    change_table :applied_coupons, bulk: true do |t|
+      t.decimal :percentage_rate, precision: 10, scale: 5
+      t.change :amount_cents, :integer, null: true
+      t.change :amount_currency, :string, null: true
```

Comment:
Rails/ReversibleMigration: change_table(with change) is not reversible. (https://rails.rubystyle.guide#reversible-migration, https://api.rubyonrails.org/classes/ActiveRecord/Migration/CommandRecorder.html)